userguide/object_detection/data-preparation.md

```diff
@@ -140,6 +140,70 @@ After you have run this script, you should have `ig02.sframe` in your current
 directory. This SFrame can be used as described in the [Object detection user
 guide](README.md).
 
+Alternately, if you don't have access to bitmasks and you're using an image 
+annotation tool like the one 
+[here](https://github.com/sgp715/simple_image_annotator), here is a code snippet
+(thanks to [@Kurry](https://github.com/Kurry)) 
+that would convert the CSV you get out of the annotator into an SFrame.
+
+```python
+import turicreate as tc
+import os
+
+IMAGES_DIR = 'images' # Change if applicable
+csv_path = 'out.csv' # assumes CSV column format is image,id,name,xMin,xMax,yMin,yMax
+csv_sf = tc.SFrame.read_csv(csv_path)
+
+def row_to_bbox_coordinates(row):
```

Comment:
This method should be updated to this:
``` python
def row_to_bbox_coordinates(row):
    """
    Takes a row and returns a dictionary representing bounding
    box coordinates:  (center_x, center_y, width, height)  e.g. {'x': 100, 'y': 120, 'width': 80, 'height': 120}
    """
    return {'x': row['xMin'] + (row['xMax'] - row['xMin'])/2, 'width': (row['xMax'] - row['xMin']),
            'y': row['yMin'] + (row['yMax'] - row['yMin'])/2, 'height': (row['yMax'] - row['yMin'])}
```
The previous code had a bug in it where `x` is the x origin and `y` is the y origin. `x` should actually be the center x, and `y` should be the center y of the bounding box. I also tried to add a comment in the documentation too because I thought this wasn't intuitive.

Reply:
Noted, thanks!

Reply:
If you read the source code, its more clear.
https://github.com/apple/turicreate/blob/master/src/unity/python/turicreate/toolkits/object_detector/util/_visualization.py#L65

Reply:
@Kurry having this error Traceback (most recent call last):
  File "classifier.py", line 32, in <module>
    info = info.unpack().rename({'X.0': 'name'})
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/turicreate/data_structures/sarray.py", line 3422, in unpack
    raise TypeError("Only SArray of dict/list/array type supports unpack")
TypeError: Only SArray of dict/list/array type supports unpack